Workbooks/Virtual Machines - Network Dependencies/Active Ports/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Active Ports",
```

Comment:
I can't tell the difference between Active Ports and Open Ports. Maybe a better name would help.

Reply:
Those were the names that our PM created them with, unfortunately changing the titles would incur changes elsewhere (e.g. blog, etc.) so will leave this be for now.